cobbler.spec

```diff
@@ -268,6 +268,15 @@ Requires(post): sed
 Web interface for Cobbler that allows visiting
 http://server/cobbler_web to configure the install server.
 
+%if 0%{?suse_version}
```

Comment:
This there a reason you're making this SUSE-specific? Also, unclosed `%if`...

Reply:
Right. This doesn't have to be suse specific… and the `if` should of course be closed.

Reply:
I removed the SUSE specific part.